src/sage/quadratic_forms/quadratic_form__reduction_theory.py

```diff
@@ -17,7 +18,7 @@ def reduced_binary_form1(self):
 
     EXAMPLES::
 
-        sage: QuadraticForm(ZZ,2,[5,5,2]).reduced_binary_form1()
+        sage: QuadraticForm(ZZ, 2, [5,5,2]).reduced_binary_form1()                      # optional - sage.symbolic
```

Comment:
Why `sage.symbolic` here while the below do not carry it?

Reply:
You are right, both of them should be `optional - sage.symbolic` for the same (bad) reason, the use of `floor` from `sage.functions`. I've made this change.

Reply:
Right. That is really bad. Perhaps we can refactor `floor` (later).